### PR TITLE
Task/improve scss compilation

### PR DESCRIPTION
**Pull request by orthagh**
Main objective : improve css compilation for dev.
Multiple developers report with master a very long compilation time for our css.
On my side, I hadn't worried as compilation take < ~5s but some have more than 20s (i saw 50s) which is unusable.

This is due to the fact phpcss cache recompile the whole chain (with bootstrap+tabler) even when one of our single file is modified.
Phpcss doesn't support (yet) caching of each import file, and is just able to cache the resulting final css file

So goals:

- [ ] permit to launch  sass + watch in cli (with dart-sass)
- [ ] `front/css.php` must use the same compiled file as the one provided by the cli
- [x] our loading paths must be the same on each side (don't load from base root of glpi)
- [ ] remove exotic syntaxes (i found, my bad, an import within a control block)
- [ ] provide a glpi/console command for watch

I'm currently blocked on @import of bootstrap within tabler (don't know why yet)
image

### CODE REVIEW

File: inc/html.class.php [hunk 1]
```diff
@@ -6517,6 +6517,8 @@ public static function compileScss($args) {
       }
 
       $scss->addImportPath(GLPI_ROOT);
+      $scss->addImportPath(GLPI_ROOT."/css");
+      $scss->addImportPath(GLPI_ROOT."/node_modules");
```
Comment (cedric-anne) [inline inc/html.class.php:6521]: This one is not OK. Indeed, required scss files are copied in `css/lib` directory to permit recompilation when using a production build of GLPI. I guess the callback declared below in `$scss->addImportPath()` have to be fixed. If some scss files are missing, they have to be added in webpack config section related to assets copy.

Comment (orthagh) [reply inc/html.class.php:6521]: i need your help on this part, the important change is we must have correct relative paths in our scss files.
Currently, we need to prefix with css which break other compilers and completion from ide

**Comment by orthagh** [comment]
To not dart sass cli doesn't support ~ syntax for modules management.
So i need a hand on webpack to keep advance on this

**Comment by orthagh** [comment]
let's close this atm, the only way to get a functionnal watch is not directly a sass cli but immplements it in our webpack.
So current changes are out of scope

**Pull request closed without merging**